packages/material-ui/src/Tabs/Tabs.test.js

```diff
@@ -538,7 +538,7 @@ describe('<Tabs />', () => {
           <Tab />
         </Tabs>,
       );
-      const tablist = getByRole('tablist');
+      const tablist = getByRole('tablist').parentNode;
```

Comment:
nit: better name for the variable. As it stands right now this is no longer the tablist but some (generic?) container.

Reply:
`const tablistContainer` in this case?

Reply:
If that is its purpose sure. Just wanted to make sure we don't introduce more confusion down the line. This was the kind of issue that made test really hard to reason about.

Reply:
Ok, that makes sense. I'm eager to explore is we can remove another intermediary level of div nesting. It would remove the need to differentiate the different nodes.